examples/Cargo.toml

```diff
@@ -16,7 +16,7 @@ env_logger = "0.10"
 log = { version = "0.4.4" }
 mio = { version = "0.8", features = ["net", "os-poll"] }
 pki-types = { package = "rustls-pki-types", version = "0.2" }
-rcgen = { version = "0.11.1", features = ["pem"], default-features = false }
```

Comment:
This is necessary to appease the minimum versions check: the examples depend on the 0.11.2+ CRL types.